core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,30 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
```

Comment:
And if the section number is added to the javadoc, who updates the javadoc in the future versions of the code when the manual changes? I would think that if you want to use the class loader context feature, then you have probably already read the section in the manual to understand how to configure it.